nym-api/src/support/storage/mod.rs

```diff
@@ -64,6 +64,7 @@ impl NymApiStorage {
         Ok(storage)
     }
 
+    #[deprecated(note = "TODO dz: obsolete because it's used for Rocket")]
     pub(crate) fn stage(storage: NymApiStorage) -> AdHoc {
```

Comment:
you might as well then remove it completely because the only thing this fairing does is to manage the state; so in your temporary rocket builder juts add explicit `rocket.manage(storage)`

Reply:
valid point, but I'll leave the function to avoid too much refactoring of rocket code, which will be removed anyway